libs/zzz/ui/src/Character/card/CharacterCardStats.tsx

```
@@ -0,0 +1,104 @@
+    'crit_dmg_',
+    'def_base',
+    'hp_base',
+  ]
+  const coreStat = [
+    ...new Set(characterStat.coreStats.flatMap(Object.keys)),
+  ][1] as (typeof coreStatMap)[keyof typeof coreStatMap]
+
+  return (
+    <FieldDisplayList bgt={bgt} sx={{ width: '100%', borderRadius: 0 }}>
+      {stats.map((stat: string) => (
+        <ListItem>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)